src/components/isearch/index.js

```diff
@@ -64,7 +64,7 @@ class ISearch extends Component {
   }
 
   render () {
-    console.log('----RENDERING----');
+    console.log('----RENDERING----', this.props.displayedItems);
```

Comment:
yeah the other idea is to use a redux action instead of console.log - then when we can turn this on and off in the redux logger.